app/com/linkedin/drelephant/spark/heuristics/StagesHeuristic.scala

```diff
@@ -192,7 +192,12 @@ object StagesHeuristic {
     }
 
     private def averageExecutorRuntimeAndSeverityOf(stageData: StageData): (Long, Severity) = {
-      val averageExecutorRuntime = stageData.executorRunTime / executorInstances
+      val allTasks : Int = if((stageData.numActiveTasks + stageData.numCompleteTasks + stageData.numFailedTasks) == 0) {
```

Comment:
Another option would be max((stageData.numActiveTasks + stageData.numCompleteTasks + stageData.numFailedTasks), 1)